README.md

```diff
@@ -95,7 +95,11 @@ $ make test
 In order to run the full suite of Acceptance tests, run `make testacc`.
 
 *Note:* Acceptance tests create real resources, and often cost money to run.
+TF_PARL is used to run all tests in parallel, TF_PARL_NUM is used to define the 
+number of paralleled tests to run, and TESTARGS is used to pass extra
+testing arguments like below.
+
 
 ```sh
-$ make testacc
+$ [TF_PARL=1] [TF_PARL_NUM=20] [TESTARGS="-run TestAccAzureRMStorage"] make testacc
```

Comment:
I think it's worth splitting this into two separate examples - one as it was (which is the default/recommended) and a second which shows an example usage in parallel (as above) - given I think most contributors will end up running these individually for cost/debugging reasons?

Reply:
Parallel running will not make user pay more, the size of test package per run is the key factor, and user can use TESTARGS to filter out necessary cases to run.

I updated the README with 3 common usage cases, hope it's helpful to end user.

Reply:
my point was more the user has the option to cancel them out; TESTARGS is a prefix match; and most tests follow the pattern:
 - `foo_basic`
 - `foo_basicUpdated`

meaning in order to run `foo_basic` using the `TESTARGS` of `-run=foo_basic` - the test `foo_basicUpdate` will also be run (leading to extra resources/costs) when in parallel, rather than being able to be cancelled via ctrl+c - it's not a big deal, but it's one to be aware of